source/uwp/UWPUnitTests/UWPUnitTests.csproj

```diff
@@ -146,8 +146,86 @@
     <Content Include="..\..\..\samples\v1.1\Elements\*.json">
       <Link>LinkedCards\v1.1\Elements\%(FileName)</Link>
     </Content>
-    <Content Include="..\..\..\samples\v1.2\Elements\*.json">
```

Comment:
Revert this here as well.

Reply:
thanks

---
In reply to: [287428525](https://github.com/microsoft/AdaptiveCards/pull/2970#discussion_r287428525) [](ancestors = 287428525)